LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -123,6 +123,8 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
                         user = await userManager.CreateUserAsync(ldapUsername).ConfigureAwait(false);
                         user.AuthenticationProviderId = GetType().FullName;
                         user.SetPermission(PermissionKind.IsAdministrator, ldapIsAdmin);
+                        user.SetPermission(PermissionKind.EnableAllFolders, LdapPlugin.Instance.Configuration.EnableAllFolders);
+                        user.SetPreference(PreferenceKind.EnabledFolders, LdapPlugin.Instance.Configuration.EnabledFolders);
```

Comment:
```suggestion
                        if (!LdapPlugin.Instance.Configuration.EnableAllFolders)
                        {
                            user.SetPreference(PreferenceKind.EnabledFolders, LdapPlugin.Instance.Configuration.EnabledFolders);
                        }
```

Enabled folders should only be set if not enabling all folders